### PR TITLE
Add proper coafile

### LINKED ISSUE #30: coafile
This is for some discussion on the `coafile`. because,   [the PR thread](https://github.com/viper-development/viper-development.github.io/pull/26) has got too long 😅 😿  

Can we go with this coafile,
```
[default]
bears = LineCountBear, FilenameBear, InvalidLinkBear, coalaBear
files = **.md, **.html, **.js, **.css, **.yml
ignore = **.min.(css|js), _site/**.(html|js|css)
max_lines_per_file = 600

[markdown]
bears = MarkdownBear, SpaceConsistencyBear, KeywordBear
files = **.md
language = markdown
use_spaces = yes

[yaml]
bears = YAMLLintBear, SpaceConsistencyBear, LineLengthBear, KeywordBear
files = **.yml
language = YAML
use_spaces = yes

[javascript]
bears = JSComplexityBear
files = **.js
ignore = _site/**.js

[html]
bears = HTMLLintBear, BootLintBear, SpaceConsistencyBear, KeywordBear
files = **.html
ignore = _site/**.html, _includes/github.html
language = HTML
use_spaces = yeah

[css]
bears = SpaceConsistencyBear
files = **.css
ignore = _site/**.css, **.min.css
use_spaces = yes
```

Log of modifications, 
- Minified css and js files, added in [css] and [js] sections
- Ignore `site` folder completely 
- CSSLintBear is posing some problems,
  - It doesn't allow more than 10 `!important`, but if !important is removed, some default bootstrap and materialize styles won't be overridden.
  - CSSLintBear's Ignore Action is not working as its supposed to, I'll start fixing it ASAP, [opened an issue too](https://github.com/coala/coala-bears/issues/1225)

### PR DESCRIPTION
**Pull request by hemangsk**
Fixes #30

### CODE REVIEW

File: .coafile [hunk 1]
```diff
@@ -1,5 +1,42 @@
...
+ignore = **.min.(css|js), _site/**
+max_lines_per_file = 600
+
+[html]
+bears = HTMLLintBear, BootLintBear, SpaceConsistencyBear, KeywordBear
+files = **.html
+ignore = _site/**, _includes/github.html
+language = HTML
 use_spaces = yeah
+htmllint_ignore = optional_tag, protocol
```
Comment (hemangsk) [inline .coafile:13]: Added this

File: .coafile [hunk 2]
```diff
@@ -1,5 +1,42 @@
...
+[css]
+bears = SpaceConsistencyBear
+files = **.css
+ignore = _site/**.css, **.min.css
+use_spaces = yes
+
+[spacing]
+bears = SpaceConsistencyBear
+files = **
+ignore = _site/**, assets/images/** , .git/**
```
Comment (hemangsk) [inline .coafile:41]: Spacing section

**Comment by hemangsk** [comment]
Ready for review 👍

**Review by sils — APPROVED** [review]

**Comment by sils** [comment]
dcd2a39 ack

**Pull request merged**